apps/docs/src/assets/stackblitz/src/styles.scss

```
@@ -1,20 +1,29 @@
-@import '@koobiq/components/prebuilt-themes/light-theme.css';
-@import '@koobiq/icons/fonts/kbq-icons';
-@import '@koobiq/design-tokens/web/css-tokens.css';
-@import '@koobiq/design-tokens/web/css-tokens-dark.css';
-@import '@koobiq/design-tokens/web/css-tokens-light.css';
+@use '@koobiq/components/prebuilt-themes/light-theme.css';
+@use '@koobiq/icons/fonts/kbq-icons';
+@use '@koobiq/design-tokens/web/css-tokens.css';
+@use '@koobiq/design-tokens/web/css-tokens-dark.css';
+@use '@koobiq/design-tokens/web/css-tokens-light.css';
+@use '@koobiq/ag-grid-angular-theme';
 
 body {
-    font-family: Inter, Arial, sans-serif;
     margin: 0;
+    font-size: var(--kbq-typography-text-normal-font-size);
```

можно через миксину задать
```suggestion
    @use '../dist/components/core/styles/common/tokens';
    @include tokens.kbq-typography-level-to-styles-css-variables(typography, text-normal);
```